src/game/client/components/chat.cpp

```diff
@@ -868,7 +868,11 @@ void CChat::OnRender()
 		int r = ((m_CurrentLine-i)+MAX_LINES)%MAX_LINES;
 		CLine& Line = m_aLines[r];
 
-		if(m_aLines[r].m_aText[0] == 0) break;
+		if(Line.m_aText[0] == 0)
+			break;
+
+		if(Line.m_ClientID && m_pClient->m_aClients[i].m_ChatIgnore)
```

Comment:
Should check if 'ClientID is >= 0' and the should use it as index for the clients, 'i' is line number

Reply:
Thanks, not sure how this didn't show up during testing